src/Model/ValueObject/Moderation/WorkflowStatus.php

```diff
@@ -25,4 +25,14 @@ public static function getAllowedValues(): array
             'DELETED',
         ];
     }
+
+    /// This method is only needed to import UDB2 items, which use
+    /// lowercase values without underscores.
+    /// It should only be used in those contexts.
+    /// It could be removed after UDB2 imports are switched off.
```

Comment:
Like @bertramakers mentioned here, this method can not be removed https://github.com/cultuurnet/udb3-silex/pull/882#discussion_r788935734

There are actually 3 distinct code flows in which an aggregate will be used in the UDB3 event sourced context:
1. When being updated through a command
2. When being loaded from the event store
3. When being replayed

The first flow typically works with a state updating method like `Offer::addLabel`. This validates internal state of the `Offer` aggregate and when all conditions are fulfilled it will persist an event `LabelAdded` by calling the `Offer::applyLabelAdded` method. It will also deliver the `LabelAdded` event to the event bus and projectors will handle the event to update the JSON representation.

The second flow that loads an aggregate calls `Offer::applyLadedAdded` directly with the persisted `LabelAdded` event.

The third flow when doing a replay takes the events from the event store and pushes them to the event listeners so that the projectors can update the JSON representation.

So `fromCultureFeedWorkflowStatus` will still be used even if the UDB2 imports are turned of (flow 1) because it will be used inside the aggregate loading (flow 2) and replay (flow 3)